fhir-persistence-jdbc/src/main/java/com/ibm/fhir/persistence/jdbc/dao/api/ResourceDAO.java

```diff
@@ -8,28 +8,23 @@
 
 import java.sql.Timestamp;
 import java.util.List;
+import java.util.Map;
 
+import com.ibm.fhir.persistence.context.FHIRPersistenceContext;
+import com.ibm.fhir.persistence.exception.FHIRPersistenceException;
+import com.ibm.fhir.persistence.exception.FHIRPersistenceVersionIdMismatchException;
+import com.ibm.fhir.persistence.jdbc.dto.Parameter;
 import com.ibm.fhir.persistence.jdbc.dto.Resource;
 import com.ibm.fhir.persistence.jdbc.exception.FHIRPersistenceDBConnectException;
 import com.ibm.fhir.persistence.jdbc.exception.FHIRPersistenceDataAccessException;
+import com.ibm.fhir.persistence.jdbc.util.SqlQueryData;
+
 
 /**
- * This Data Access Object interface provides methods creating, updating, and retrieving rows in the FHIR Resource table.
- * @author markd
```

Comment:
seems we need a rule about how to change the author info, either remove the old author or add the new author to the list or simply don't keep any author info to the codes ... when I modify existing files, I don't add my name to the author list, not sure if this is a bad practice...

Reply:
same

Reply:
we just discussed this as a team and agreed that the `@author` tags add limited value beyond what git history already provides. furthermore, they get stale over time and become maintenance burden.  we will remove them incrmentally over time (as we edit classes) and this PR is consistent with that approach